### ISSUE TITLE
Background color for new forum posts in dark/black mode

### ISSUE THREAD

**Issue by bmcutler** [Submitty/Submitty#6543]
**Describe the bug**
new posts in dark mode should be highlighted with a background color.
(In light mode they are light blue vs white for old posts)

Also, when modified, please update documentation:
https://submitty.org/student/communication/forum
